### DISCUSSION TITLE
Newly installed instance of Next.js exits after output of started on localhost:3000

### DISCUSSION THREAD

**Discussion by arsekil** [vercel/next.js#47548] [Help] [answered]
### Summary

I am using the latest version of next.js, no `src` or `app` directories.
It simply kicks me out after writing in that is started on port 3000. There are no error messages.

I checked my other project of next.js and it's working fine.

Did anyone encounter such problem?

Thanks.

**Comment by icyJoseph** [comment, marked as answer]
Hi, 

Could you double check that your have a Node.js version matching your OS? `node -p 'process.arch'` run that command in your terminal, it should output your system's architecture.

If that doesn't work, then re-install Node.js to a matching version. 

I've seen this happen to Windows users quite a lot.

Otherwise we'll need more info about your system and project.

**Reply by arsekil** [reply]
Hi, @icyJoseph 

I figured I upgraded my Node to a 32-bit version, but my OS is 64bits. I think this might jave been the problem, right?

Now it's working great.

Thanks for pointing it out.

Solved.

**Reply by icyJoseph** [reply]
Awesome! Please don't forget to mark as answered. This is happening to quite a few people.
